cmd/svcat/binding/bind_cmd.go

```diff
@@ -126,6 +143,12 @@ func (c *bindCmd) bind() error {
 		return err
 	}
 
+	if c.wait {
+		glog.V(2).Info("Waiting for instance to be bound...")
```

Comment:
nit, do with this as you wish:  I dislike the idea of an instance being bound.  It sounds singular to me, I prefer the notion of creating a new binding.  If you don't agree, ignore my comment :)

Reply:
I changed it to `Waiting for binding to be injected...`. I can't say "binding to be created" because it exists immediately, and we are really just waiting for it be usable.